### PR TITLE
Add split screen example

### PR DESCRIPTION
**Pull request by arthuro555**
### Description

Shows a simple way to make split-screen multiplayer in GDevelop, using the behavior remapper extension for giving custom controls to each player and the render to sprite extension to render each player's camera into a sprite. Each player also has a simple randomly colored tag with its ID to help differentiate them.

### Demo

https://games.gdevelop-app.com/game-ce7eaaae-f4b2-400d-9448-13c03106fee5/index.html

Controls: Arrow key+Space, WASD+E, TFGH+Z, IJKL+O.

### Download as zip

[split-screen.zip](https://github.com/GDevelopApp/GDevelop-examples/files/7695543/split-screen.zip)

## Checklist

- [x] I've followed all of [the best practices](http://wiki.compilgames.net/doku.php/gdevelop5/community/guide-for-submitting-an-example).
- [x] My game has a proper name in the game properties. 
- [x] My game package name behind with `com.example.`.
- [x] My game has all events unfolded.
- [x] I've added myself as the author in the game properties.
- [x] I've included a file called "README.md" with a description in proper English, explaining what this example is doing.
- [x] I confirm that this game and all of its resources can be integrated to this GitHub repository, distributed and MIT licensed.
- [x] I've cleaned unused resources.

### CODE REVIEW

File: examples/split-screen/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Split screen multiplayer in two events.
```
Comment (Midhil457) [inline examples/split-screen/README.md:1]: Maybe have it as `A split screen multiplayer made in just two events.`

File: examples/split-screen/layouts/main.json [hunk 2]
```diff
@@ -0,0 +1,1020 @@
+{
```
Comment (Midhil457) [inline examples/split-screen/layouts/main.json:1]: Correct me if I am wrong. I don't think Multiple Files for saving won't work for examples. Only single file saving

File: examples/split-screen/layouts/new-scene.json [hunk 3]
```diff
@@ -0,0 +1,1020 @@
+{
```
Comment (Midhil457) [inline examples/split-screen/layouts/new-scene.json:1]: Is this scene used? If not, please remove it :)

File: examples/split-screen/game.json [hunk 4]
```diff
@@ -0,0 +1,149 @@
+{
```
Comment (Midhil457) [inline examples/split-screen/game.json:1]: Could we have a text on each view with the corresponding player's name

**Review by Midhil457 — CHANGES_REQUESTED** [review]

**Comment by Bouh** [comment]
- As @Midhil457 said before, only the example in a unique file can be used.
- Maybe just unlock all objects before merging.
- `game.json` have to be same as the folder name.
- thumbnail.png is missing

**Comment by arthuro555** [comment]
> As Midhil457 said before, only the example in a unique file can be used.

Oh, really? I would have sworn that I had used a multiple files project when making the firebase example 🤔

File: examples/split-screen/layouts/new-scene.json (hunk 3 above)
Comment (arthuro555) [reply examples/split-screen/layouts/new-scene.json:1]: Ah, pesky GDevelop not deleting old scene files when renaming them

File: examples/split-screen/game.json (hunk 4 above)
Comment (arthuro555) [reply examples/split-screen/game.json:1]: Sorry I don't understand your request

**Comment by arthuro555** [comment]
> thumbnail.png is missing

That's a necessity now? I am extremely bad at graphics, I can't make a thumbnail.

> Maybe unlock all objects before merging, let's people move sprite and understand how works the screen objects.

Anyone can unlock objects, is it really necessary? Personally I like objects like that one that don't need to and shouldn't be moved for changing the level locked.

File: examples/split-screen/game.json (hunk 4 above)
Comment (Bouh) [reply examples/split-screen/game.json]: Done 
image

**Review by Bouh — APPROVED** [review]

File: examples/split-screen/game.json (hunk 4 above)
Comment (arthuro555) [reply examples/split-screen/game.json]: Ah, that's nice indeed, thanks :)

**Comment by arthuro555** [comment]
@Midhil457 If you don't have any more comments on this I'll merge this tomorrow :)

**Comment by Midhil457** [comment]
DId some small changes:
- Moved the plaforming area to the bottom of the screens
- Make the camera zoom to make the platformer visible

Approving this!

**Review by Midhil457 — APPROVED** [review]

**Pull request merged**